server/chat/backend/agent/orchestrator/triage.py

```
@@ -0,0 +1,163 @@
+        if len(decision.inputs) > _MAX_SUBAGENTS:
+            logger.warning(
+                "triage: LLM produced %d inputs > cap %d — truncating",
+                len(decision.inputs), _MAX_SUBAGENTS,
+            )
+            decision.inputs = decision.inputs[:_MAX_SUBAGENTS]
+
+        logger.info(
+            "triage: incident %s -> mode=%s sub-agents=%d",
+            incident_id_hash, decision.mode, len(decision.inputs),
+        )
+        return decision
```

Ruff (0.15.12)

[warning] Consider moving this statement to an `else` block

(TRY300)